### ISSUE TITLE
partially filled hitbtc order has incorrect quantity

### ISSUE THREAD

**Issue by npomfret** [timmolter/XChange#1645]
My order was partially filled.  The original quantity was 0.4, the website shows this:

_Exec./Total Amount (ETH)
0.048/0.400_

The implementation says I have 352 remaining!? The api returns this:

    {
      "orderId": "1472765469",
      "orderStatus": "partiallyFilled",
      "lastTimestamp": 1501245842123,
      "orderPrice": "0.068140",
      "orderQuantity": 352,
      "avgPrice": "0.068122",
      "quantityLeaves": 352,
      "type": "limit",
      "timeInForce": "GTC",
      "cumQuantity": 48,
      "clientOrderId": "B_ETHBTC_1501245841954",
      "symbol": "ETHBTC",
      "side": "buy",
      "execQuantity": 48
    }

I guess the units are wrong, it looks like they need to be divided by 1000? Can someone confirm?  And is this the same for all markets?

**Comment by npomfret** [comment]
Ah, is this what the lot sizes are for?

**Comment by cymp** [comment]
Yes, that's one of the reasons why this can be a nightmare sometimes, cf https://github.com/timmolter/XChange/issues/1633

**Comment by npomfret** [comment]
Right.  Moving to the new api sounds like a good idea.  In the mean time I'll try and fix this.

**Comment by cymp** [comment]
Did you get a chance to work on this npomfret?

**Comment by npomfret** [comment]
not yet, sorry
